pkg/solana/logpoller/log_poller.go

```diff
@@ -382,3 +382,22 @@ func (lp *Service) backgroundWorkerRun(ctx context.Context) {
 func (lp *Service) FilteredLogs(ctx context.Context, queryFilter []query.Expression, limitAndSort query.LimitAndSort, queryName string) ([]Log, error) {
 	return lp.orm.FilteredLogs(ctx, queryFilter, limitAndSort, queryName)
 }
+
+// Where is a query.Where wrapper that ignores the Key and returns a slice of query.Expression rather than
+// query.KeyFilter. If no expressions are provided, or an error occurs, an empty slice is returned.
+func Where(expressions ...query.Expression) ([]query.Expression, error) {
+	filter, err := query.Where(
+		"",
+		expressions...,
+	)
+
+	if err != nil {
+		return []query.Expression{}, err
+	}
+
+	if filter.Expressions == nil {
+		return []query.Expression{}, nil
+	}
+
+	return filter.Expressions, nil
+}
```

Comment:
Shouldn't this Where helper go in `query.go` or `parser.go`, rather than `log_poller.go`?

Reply:
Looks like `query` would make the most sense, since all of the types it uses start with `query.`

Reply:
Wait, actually why is this even in the chainlink-solana repo? Looks like a chain-agnostic helper that belongs in `chainlink-common`

Reply:
It probably does belong in common, but that will take longer to add.

Reply:
Alright, we can skip it for now.